e2e/testcases/helm_sync_test.go

```diff
@@ -136,6 +138,185 @@ func TestPublicHelm(t *testing.T) {
 	}
 }
 
+// TestHelmWatchConfigMap can run on both Kind and GKE clusters.
+// It tests that helm-sync properly watches ConfigMaps in the RSync namespace and pulls updated values.
+func TestHelmWatchConfigMap(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.SyncSource, ntopts.Unstructured)
+
+	configMapFilePath := "../testdata/root-sync-helm-cm-update/configmap-1.yaml"
+	nt.T.Logf("Apply the ConfigMap with values to the cluster defined in %s", configMapFilePath)
+	nt.MustKubectl("apply", "-f", configMapFilePath)
```

Comment:
Echoing my comment on the testdata files, it would be preferred to use the go client for the types of operations in these tests as opposed to shelling out to kubectl. It also makes it hard to follow what's happening in the test since the contents of the configmaps are declared elsewhere

Reply:
Done